### DISCUSSION TITLE
integrate strapi and Angular application with wysiwyg editor

### DISCUSSION THREAD

**Discussion by Roman7425** [strapi/strapi#6774] [Legacy Topics]
I need to integrate the strapi API with an application on Angular. More precisely, I need to integrate the wysiwyg editor in this application with the strapi API. How can i do this?

**Comment by mahaveer0496** [comment]
Like you would do with any other API. Render a WYSIWYG editor (like TinyMCE) in your angular frontend and make API requests to strapi to interact with the data.
